java/client/src/main/java/org/apache/rocketmq/client/java/impl/consumer/ConsumerImpl.java

```diff
@@ -123,7 +123,10 @@ protected ListenableFuture<ReceiveMessageResult> receiveMessage(ReceiveMessageRe
     }
 
     private AckMessageRequest wrapAckMessageRequest(MessageViewImpl messageView) {
-        final Resource topicResource = Resource.newBuilder().setName(messageView.getTopic()).build();
+        final Resource topicResource = Resource.newBuilder()
+            .setResourceNamespace(clientConfiguration.getNamespace())
```

Comment:
Does `namespace` make sense for consumer group?

Reply:
Yes, for sure.

Reply:
Group's namespace is added in getProtobufGroup() method.